bindings/go/mio/mio.go

```diff
@@ -40,7 +40,6 @@
 package mio
 
 // #cgo CFLAGS: -I/usr/include/motr
-// #cgo CFLAGS: -I../../.. -I../../../extra-libs/galois/include
```

Comment:
Please validate that you are able to build and execute mio.go and mkv.go after this update to the CFLAGS.

Reply:
This (CFLAGS) section is commented in both files. So removing the Galois part from CFLAGS will not affect its compilation.

Reply:
Let's ask @andriytk to verify this.

Reply:
It's not commented out, it's how one can insert C-code directly into Go-code.

Please remove only galois-related -I-values and leave the rest as it is.

Reply:
I am able to build the code.
    

> [root@ssc-vm-rhev4-1085 fork-cortx-motr]# cd bindings/go/mcp/
>     [root@ssc-vm-rhev4-1085 mcp]# go build
>     [root@ssc-vm-rhev4-1085 mcp]# ll
>     total 2740
>     -rwxr-xr-x 1 root root 2795672 Sep 17 02:29 mcp
>     -rw-r--r-- 1 root root    4194 Sep  6 07:13 mcp.go
>     [root@ssc-vm-rhev4-1085 mcp]#
>

Reply:
You probably have Motr rpm installed, so that `-I/usr/include/motr` works in your case. Try to uninstall motr rpm and build.

Reply:
No I dont have motr rpm installed. I only have compiled motr code for this branch.

> [root@ssc-vm-rhev4-1085 mcp]# rpm -qa | grep motr
> [root@ssc-vm-rhev4-1085 mcp]# rpm -qa | grep cortx
> [root@ssc-vm-rhev4-1085 mcp]# ll /usr/include/ | grep motr
> [root@ssc-vm-rhev4-1085 mcp]# ll /usr/include/m
> malloc.h    math.h      mcheck.h    memory.h    menu.h      misc/       mntent.h    monetary.h  mqueue.h    mtd/
> [root@ssc-vm-rhev4-1085 mcp]#
>

Reply:
Well, it's strange, but it works indeed for me as well even with this:

```
--- a/bindings/go/mio/mio.go
+++ b/bindings/go/mio/mio.go
@@ -39,9 +39,6 @@
 // For the usage example, refer to mcp utility.
 package mio
 
-// #cgo CFLAGS: -I/usr/include/motr
-// #cgo CFLAGS: -I../../.. -I../../../extra-libs/galois/include
-// #cgo CFLAGS: -DM0_EXTERN=extern -DM0_INTERNAL=
 // #cgo CFLAGS: -Wno-attributes
 // #cgo LDFLAGS: -L../../../motr/.libs -Wl,-rpath=../../../motr/.libs -lmotr
 // #include <stdlib.h>
```

Reply:
Ok, let's drop these lines.

Reply:
As per discussion with @andriytk, it is decided to drop only `-I../../../extra-libs/galois/include` for now, because removing other lines from `mio.go` and `mkv.go` caused compilation failures.